bpf/lib/common.h

```diff
@@ -220,13 +221,21 @@ struct lb4_service {
 	__u16 port;
 	__u16 count;
 	__u16 rev_nat_index;
+	__u16 weight;
 } __attribute__((packed));
 
 struct lb4_reverse_nat {
 	__be32 address;
 	__u16 port;
 } __attribute__((packed));
 
+/* Must match lbmap.go maxSeq */
+#define LB_RR_MAX_SEQ 31
+struct lb_sequence {
+	__u16 count;
+	__u16 idx[LB_RR_MAX_SEQ];
```

Comment:
I think we should leave the go side as is,  but have it generate the BPF define. That way if we need to change the len we simply change maxSeq, rebuild, restart the daemon and have it generate the BPF side for us. We are already doing this for prefixes etc and this is no different. 

It does not add a lot of value in hacking the makefile for the project to set a define for lb.